buildenv/jenkins/testJobTemplate

```diff
@@ -302,8 +302,14 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							}
 							stringParam('BUILD_LIST', "${ACTUAL_BUILD_LIST}", "Specific test directory to compile, set blank for all projects to be compiled")
 							stringParam('TARGET', "${ACTUAL_TARGET}", "Test TARGET to execute")
-							stringParam('CUSTOM_TARGET', "", "Only used when the custom target is specified in TARGET , e.g. jdk_custom, langtools_custom, etc., CUSTOM_TARGET=path to the test class to execute")
-							
+							stringParam('CUSTOM_TARGET', "", '''
+								Only used when the custom target is specified in TARGET , e.g. jdk_custom, langtools_custom, system_custom, etc.,<br>
+								Example 1:<br>
+								TARGET=system_custom<br>
+								CUSTOM_TARGET=test/jdk/java/math/BigInteger/LargeValueExceptions.javaCUSTOM_TARGET=path to the test class to execute<br>
```

Comment:
Can move the end of 309 (CUSTOM_TARGET=path to the test class to execute) up to the end of line 306.  The CUSTOM_TARGET examples are mixed up (the jdk_custom one is listed under example 1, and the system_custom one is listed in example 2).  Need to swap line 309 and line 312.